pkg/cmd/roachtest/run.go

```diff
@@ -439,6 +449,41 @@ func maybeDumpSummaryMarkdown(r *testRunner) error {
 	return nil
 }
 
+func maybeEmitDatadogEvent(
+	client *statsd.Client, opSpec *registry.OperationSpec, eventType ddEventType, operationID uint64,
+) {
+	if client == nil {
+		return
+	}
+	status := "started"
+
+	switch eventType {
+	case eventOpStarted:
+		status = "started"
+	case eventOpRan:
+		status = "finished running; waiting for cleanup"
+	case eventOpFinishedCleanup:
+		status = "cleaned up its state"
+	case eventOpError:
+		status = "ran with an error"
+	}
+	ev := statsd.NewEvent(fmt.Sprintf("op %s %s", opSpec.Name, status), "")
```

Comment:
Are we able to see on which cluster this operation ran? Or is that configured somewhere else?

Reply:
Should we throw that in the event's details? We could do that. Currently you'd see what "cluster" it ran *from* (as that's the one generating the event), but not the one it ran on.

Reply:
Done.